recipe/meta.yaml

```diff
@@ -30,24 +52,29 @@ build:
     - '*'
 
 outputs:
-  - name: sysroot_{{ target_platform }}
+  - name: sysroot_{{ cross_target_platform }}
     script: build-sysroot.sh
     build:
       noarch: generic
+    requirements:
+      run_constrained:
+        - __glibc >={{ version }},<3.0.a0
```

Comment:
Why do we need this constraint?

Reply:
If we build packages using 2.17, will they work in machines that locally have an earlier version?

Reply:
No, but then we need a `run_exports` not a `run_constrained`.

Reply:
But this is a CDT. It doesn’t ever appear in an env at runtime. It is just used for building.

Reply:
We link at runtime to the local copy of glibc, right?

Reply:
The virtual package is a constraint on this local version.

Reply:
> It doesn’t ever appear in an env at runtime.

Then what is the use of this `run_constrained`? This only prevents users from cross compiling.

Reply:
Ohhhhhhh. You mean a run export of the virtual package constraint, not of the cdt. Facepalm. Yes that will work but ppl with conda < 4.8 will have trouble.

Reply:
Sure. That's okay in my opinion. If they want new things, use newer conda.